gm4_calling_bell/data/gm4_calling_bell/functions/day_clock.mcfunction

```diff
@@ -0,0 +1,4 @@
+
+# check day, daily
+execute store result score $day gm4_calling_bell run time query day
+schedule function gm4_calling_bell:day_clock 120s
```

Comment:
This can get a bit behind when players sleep, right? But presumably it's not that big of a problem.

Reply:
Correct, I will reduce the clock though